src/Icons/README.md

```diff
@@ -0,0 +1,73 @@
+# Symfony UX Icons
+
+## Installation
+
+```bash
+composer require symfony/ux-icons
+```
+
+## Add Icons
+
+No icons are provided by this package. Add your svg icons to the `templates/icons/` directory.
+The name of the file is used as the name of the icon (`name.svg` will be named `name`).
+
+When icons are rendered, any attributes (except `viewBox`) on the file's `<svg>` element will
+be removed. This allows you to copy/paste icons from sites like
+[heroicons.com](https://heroicons.com/) and not worry about hard-coded attributes interfering with
+your design.
+
+## Usage
+
+```twig
+{{ ux_icon('user-profile', {class: 'w-4 h-4'}) }} <!-- renders "user-profile.svg" -->
+
+{{ ux_icon('sub-dir:user-profile', {class: 'w-4 h-4'}) }} <!-- renders "sub-dir/user-profile.svg" (sub-directory) -->
+```
+
+### HTML Syntax
+
+> [!NOTE]
+> `symfony/ux-twig-component` is required to use the HTML syntax.
+
+```html
+<twig:UX:Icon name="user-profile" class="w-4 h-4" /> <!-- renders "user-profile.svg" -->
+
+<twig:UX:Icon name="sub-dir:user-profile" class="w-4 h-4" /> <!-- renders "sub-dir/user-profile.svg" (sub-directory) -->
+```
+
+## Caching
+
+To avoid having to parse icon files on every request, icons are cached (`cache.system` by
+default but can be [configured](#full-default-configuration)).
```

Comment:
Do we need to allow this to be configured? It feels like allowing the "validation metadata cache" to be configured.

Reply:
The reason I made it configurable is: if you have a ton of icons (1000's), the container warmup can be slow (ie 2-3 seconds). If we're ok with that, I can remove this config and the warmup command.